app/controllers/SampleController.scala

```diff
@@ -51,10 +51,13 @@ class SampleController @Inject() (
   protected def chooseStatusView(c: Case, notFilledForm: Form[Option[SampleStatus]], options: Option[String])(
     implicit request: AuthenticatedRequest[_]
   ): Html =
-    if (options.contains("liability"))
+    if (options.contains("liability")) {
       views.html.change_liablity_sending_sample(c, notFilledForm)
-    else
+    } else if (options.contains("correspondence")) {
+      views.html.change_correspondence_sending_sample(c, notFilledForm)
+    } else {
       views.html.change_sample_status(c, notFilledForm)
+    }
```

Comment:
Not necessarily something we necessarily need to change in this PR, but do you think we need this concept of `options` at all? It seems like we could just match on the `` c.application.`type` `` here

Reply:
Good spot!. We have changed it now to use application.'type' instead of options.